resharper/resharper-unity/src/CSharp/Daemon/Stages/BurstCodeAnalysis/BurstCodeAnalysisUtil.cs

```diff
@@ -47,6 +47,15 @@ public static bool IsBurstPermittedType([CanBeNull] IType type)
             }
         }
 
+        public static bool IsQualifierOpenType(IInvocationExpression invocationExpression)
+        {
+            var result =
+                ((invocationExpression.InvokedExpression as IReferenceExpression)?.QualifierExpression as
```

Comment:
please split to several lines with variable declartions, it is hard to read that code. Why are you checking qualifier only for IReferenceExpression?